CMakeLists.txt

```diff
@@ -0,0 +1,27 @@
+cmake_minimum_required(VERSION 3.16.3...3.19.7 FATAL_ERROR)
+
+project(SlicerLiteExtension)
+
+#-----------------------------------------------------------------------------
+# Extension meta-information
+set(EXTENSION_HOMEPAGE "https://www.slicer.org/wiki/Documentation/Nightly/Extensions/SlicerLiteExtension")
+set(EXTENSION_CATEGORY "Examples")
+set(EXTENSION_CONTRIBUTORS "Laurenn Lam (Kitware SAS), Thibault Pelletier (Kitware SAS), Julien Finet (Kitware SAS)")
+set(EXTENSION_DESCRIPTION "Slicer extension with a simple UI to load/visualize/segment your dicom data")
+set(EXTENSION_ICONURL "https://www.example.com/Slicer/Extensions/SlicerLiteExtension.png")
```

Comment:
Change to https://github.com/KitwareMedical/SlicerLiteExtension/raw/main/Icons/... URL and add Icon (maybe placeholder)